zipline-gradle-plugin/src/test/kotlin/app/cash/zipline/gradle/ZiplineCompileTaskTest.kt

```diff
@@ -41,7 +41,28 @@ class ZiplineCompileTaskTest {
   @Test
   fun `write to and read from zipline`() {
     val rootProject = File("src/test/projects/happyPath")
-    assertZiplineCompileTask(rootProject, true)
+    assertZiplineCompileTask(rootProject) {
+      val readZiplineFile = it.source().buffer().use { source ->
+        ZiplineFile.read(source)
+      }
+      assertEquals(CURRENT_ZIPLINE_VERSION, readZiplineFile.ziplineVersion)
+      quickJs = QuickJs.create()
+      quickJs!!.execute(readZiplineFile.quickjsBytecode.toByteArray())
+      val exception = assertFailsWith<Exception> {
+        quickJs!!.evaluate("demo.sayHello()", "test.js")
+      }
+      // .kt files in the stacktrace means that the sourcemap was applied correctly.
+      assertThat(exception.stackTraceToString()).startsWith(
```

Comment:
This is repeated with the ZiplineCompiler test. I suspect the nicest way to structure this is for this test to be even more on the happy path – maybe confirm that a sayHello() program returns "hello" ??